package.json

```diff
@@ -644,6 +644,11 @@
           "markdownDescription": "Enable the [ReplaceWithRegister](https://github.com/vim-scripts/ReplaceWithRegister) plugin for Vim.",
           "default": false
         },
+        "vim.smartRelativeLine": {
+          "type": "boolean",
+          "markdownDescription": "`#editor.lineNumbers#` is determined by the active Vim mode.",
```

Comment:
Could you be a bit more specific in the description, to make it clear that it'll be absolute in Insert mode and relative otherwise?

Reply:
Rebased and updated.